src/module/address/application/service/address.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable, Inject, NotFoundException } from '@nestjs/common';
+import { IAddressRepository } from '../repository/address.repository.interface';
+import { Address } from '../../domain/address.entity';
+import { CreateAddressDto, UpdateAddressDto } from '../dto/address.dto';
+
+@Injectable()
+export class AddressService {
+  constructor(
+    @Inject('ADDRESS_REPOSITORY') 
```

Comment:
whe should be using a const called ADDRESS_REPOSITORY instead of the string. 
in the  file address.repository.interface.ts  should exists
`export const  ADDRESS_REPOSITORY = "ADDRESS_REPOSITORY"`